twill-core/src/main/java/org/apache/twill/internal/utils/LogLevelUtil.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.twill.internal.utils;
+
+import com.google.common.collect.Maps;
+import org.apache.twill.api.logging.LogEntry;
+import org.apache.twill.internal.Constants;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+/**
+ * Collections of helper functions for log level change.
+ */
+public final class LogLevelUtil {
+
+  /**
+   * Convert the log level argument value type to string.
+   */
+  public static Map<String, String> convertLogLevelValuesToString(Map<String, LogEntry.Level> logLevels) {
+    return Maps.transformEntries(logLevels, new Maps.EntryTransformer<String, LogEntry.Level, String>() {
+      @Override
+      public String transformEntry(String loggerName, LogEntry.Level level) {
+        return level.name();
+      }
+    });
+  }
+
+  /**
+   * Convert the log level argument type to LogEntry.Level
+   */
+  public static Map<String, LogEntry.Level> convertLogLevelValuesToLogEntry(Map<String, String> logLevels) {
+    return Maps.transformEntries(logLevels, new Maps.EntryTransformer<String, String, LogEntry.Level>() {
+      @Override
+      public LogEntry.Level transformEntry(String loggerName, String level) {
+        return LogEntry.Level.valueOf(level);
+      }
+    });
+  }
+
+  /**
+   * Get the log level arguments for a twill runnable.
+   *
+   * @param runnableName name of the runnable.
+   * @param logLevels the arguments for all runnables.
+   * @return the map of the log level arguments for the runnable, empty if there is no argument.
+   */
+  public static Map<String, LogEntry.Level> getLogLevelForRunnable(
```

Comment:
We shouldn't have this method as we shouldn't use `LOG_ALL_RUNNABLES` as a special runnable name. It's better to have the log levels populated to each runnable log level map when setting/changing it.